controls/radrichtextbox/getting-started/getting-started-active-document-editor.md

```diff
@@ -0,0 +1,62 @@
+---
+title: Active Document Editor
+page_title: Active Document Editor
+description: Active Document Editor
+slug: radrichtextbox-getting-started-activedocumenteditor
+tags: idocumenteditor,activedocumenteditor,activedocumenteditorchanged
+published: True
+position: 6
+---
+
+# Active Document Editor
+
+RadRichTextBox displays the different parts of its content using few different document editors. A document editor is a control that implements the IDocumentEditor interface. For example the RadRichTextBox control.
+
+## Document Editor Types
+
+There are few types of document editors.
+
+* __MainDocument__: This is the RadRichtTextBox control that displays the main document.
+
+* __Header__: This is the document editor that displays a header in the document. When you start editing a header, this type of document editor is activated. Read more about headers and footers in [Headers and Footers help article]({%slug radrichtextbox-features-headers-and-footers%})
+
+* __Footer__: This is the document editor that displays a footer in the document. When you start editing a footer, this type of document editor is activated. 
+
+* __Note__: This is the document editor that displays notes in the document. When you start editing a note, this type of document editor is activated. Read more about notes in the [Footnotes and Endnotes]({%slug radrichtextbox-features-footnotes-endnotes%}) help article.
+
+* __Comment__: This is the document editor that displays a comment in the document. When you start editing a comment, this type of document editor is activated. Read more about comments in the [Comments]({%slug radrichtextbox-features-comments%}) help article.
+
+Currently, all document editors are of type RadRichTextBox. In other words, RadRichTextBox uses child RadRichTextBox controls as IDocumentEditors to edit the different parts of its content.
+
+## Using the Active Document Editor
+
+The active editor can be get via the __ActiveDocumentEditor__ property of RadRichTextBox.
```

Comment:
"You can obtain the active editor via the __ActiveDocumentEditor__ property of RadRichTextBox." sounds better to me